admin-data-explorer-play-server/src/main/scala/net/wiringbits/webapp/utils/admin/repositories/DatabaseTablesRepository.scala

```diff
@@ -62,7 +62,8 @@ class DatabaseTablesRepository @Inject() (database: Database)(implicit
   def find(tableName: String, primaryKeyValue: String): Future[Option[TableData]] = Future {
     database.withTransaction { implicit conn =>
       val settings = tableSettings.unsafeFindByName(tableName)
-      val maybe = DatabaseTablesDAO.find(tableName, settings.primaryKeyField, primaryKeyValue)
+      val primaryKeyType = settings.primaryKeyDataType
+      val maybe = DatabaseTablesDAO.find(tableName, settings.primaryKeyField, primaryKeyValue, primaryKeyType)
```

Comment:
I'd suggest t use named arguments to avoid positional-argument issues, for example, sending `primaryKeyValue` while `primaryKeyType` is expected.

Reply:
Like this?
```
val maybe = DatabaseTablesDAO.find(
    tableName = tableName,
    primaryKeyField = settings.primaryKeyField,
    primaryKeyValue = primaryKeyValue,
    primaryKeyType = primaryKeyType
)
```

Reply:
Exactly